src/main/java/com/google/api/codegen/transformer/py/PythonPackageMetadataTransformer.java

```diff
@@ -267,6 +316,8 @@ private OptionalArrayMethodView createExampleApiMethodView(GapicMethodContext co
         namespacePackages.add(subPackage);
       }
     }
+    namespacePackages.add("google.cloud.proto");
```

Comment:
Google.cloud.gapic was switched over to the new path "google.cloud.api_vx.gapic" and that should be computed already.